### DISCUSSION TITLE
Centering Bands on the Ribbon

### DISCUSSION THREAD

**Discussion by MichaelWDenney** [kirill-grouchnikov/radiance#493] [Q&A] [answered]
Is there a setting to have the bands within a ribbon centered vertically rather than always to the left?  I suspect there is not and I don't think I would like the result, but this was asked by our QA.

**Comment by kirill-grouchnikov** [comment, marked as answer]
Centered vertically or horizontally? No setting for both, and no plans to add that. You can "experiment" with your own UI in any tool (photoshop, figma, ...) by cut-pasting elements in your screenshots to see how it might look like, but again - no plans to add such a configuration option.

**Reply by MichaelWDenney** [reply]
Sure.  I just wanted to make sure it wasn't supported before I shot down the suggestion.  To be clear, they were suggesting something like:
![image](https://github.com/user-attachments/assets/5c6f5bf0-3f8c-4d8a-8637-48ed340e7cb5)

The thinking was that could be used to keep the bands in a sparsely populated ribbon closer to the ribbon title.  But, in my opinion, that looks stranger than the alternative of keeping everything to the left.
